packages/design-system/src/components/mediaInput/index.js

```diff
@@ -221,6 +236,7 @@ export const MediaInput = forwardRef(function Media(
               internalRef.current = node;
             }}
             id={buttonId}
+            $showImage={variant !== MEDIA_VARIANTS.NONE}
```

Comment:
Nit: in theory, this doesn't need to be a transient prop, it doesn't belong to the list [here](https://github.com/emotion-js/emotion/blob/main/packages/is-prop-valid/src/props.js)

See also: https://xwp.slack.com/archives/CCCAUAH6F/p1643891495809269

Curious -- did it show up in DOM despite not being in the list or it was added just in case?

Reply:
Ah nope, that was a misstep by me. It does not show up in the dom
<img width="210" alt="image" src="https://user-images.githubusercontent.com/22185279/156213766-15a70fff-d6ad-4750-b316-6c7cdfcaf82a.png">

Nice catch!